styles/pages/diary.scss

```diff
@@ -39,15 +39,21 @@
 
   &__arrows {
     line-height: 24px;
+    width: 80px;
 
-    &--left {
-      margin-right: 5px;
+    &--top {
+      //
```

Comment:
?

Reply:
Well right now **top arrows** do not really need any specific css rules, but the class is in JS so I would like to keep it css file as well. For better understanding of the structure, I'd say.